<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SangMike-PC" name="com.guru.testcases.LoginTest" tests="3" failures="0" timestamp="18 Apr 2017 16:42:28 GMT" time="31.322" errors="1">
  <testcase name="TC_04_TransferMoneyIntoCurrentAccount" time="7.891" classname="com.guru.testcases.LoginTest"/>
  <testcase name="TC_03_AddNewAccountAndGetAccountID" time="5.946" classname="com.guru.testcases.LoginTest"/>
  <testcase name="TC_01_CreateNewCustomerAccountAndGetID" time="17.485" classname="com.guru.testcases.LoginTest">
    <error type="java.lang.AssertionError" message="Layout test failed">
      <![CDATA[java.lang.AssertionError: Layout test failed
	at org.testng.Assert.fail(Assert.java:94)
	at common.AbstractPage.checkUIpage(AbstractPage.java:82)
	at demoGuruPages.HomePage.checkUI(HomePage.java:22)
	at com.guru.testcases.LoginTest.TC_01_CreateNewCustomerAccountAndGetID(LoginTest.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- TC_01_CreateNewCustomerAccountAndGetID -->
</testsuite> <!-- com.guru.testcases.LoginTest -->
